docs/source/user/aerodyn/examples/ad_primary_example.dat

```diff
@@ -19,26 +17,36 @@ False         CompAA             - Flag to compute AeroAcoustics calculation [us
 "default"     SpdSound           - Speed of sound in working fluid (m/s)
 "default"     Patm               - Atmospheric pressure (Pa) [used only when CavitCheck=True]
 "default"     Pvap               - Vapour pressure of working fluid (Pa) [used only when CavitCheck=True]
-======  Blade-Element/Momentum Theory Options  ====================================================== [unused when WakeMod=0 or 3]
-          1   SkewMod            - Type of skewed-wake correction model (switch) {1=uncoupled, 2=Pitt/Peters, 3=coupled} [unused when WakeMod=0 or 3]
-"default"     SkewModFactor      - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when SkewMod=2; unused when WakeMod=0 or 3]
-f             TipLoss            - Use the Prandtl tip-loss model? (flag) [unused when WakeMod=0 or 3]
-f             HubLoss            - Use the Prandtl hub-loss model? (flag) [unused when WakeMod=0 or 3]
+======  Blade-Element/Momentum Theory Options  ====================================================== [unused when WakeMod=0 or 3, except for BEM_Mod]
+          2   BEM_Mod            - BEM model {1=legacy NoSweepPitchTwist, 2=polar} (switch) [used for all Wake_Mod to determine output coordinate system]
+--- Skew correction
+          1   Skew_Mod           - Skew model {0=No skew model, -1=Remove non-normal component for linearization, 1=skew model active}
```

Comment:
What is the difference between Skew_Mod = 0 and Skew_Mod = 1 with SkewModCorr = False and SkewRedistr_Mod = 0?

Reply:
Currently, they would be the same. Skew_Mod is like a master switch.

Reply:
Makes sense.  I suggest adding a sentence in the input file documentation to clarify this point.